package.json

```diff
@@ -10,9 +10,7 @@
     "dist"
   ],
   "dependencies": {
-    "@types/node-fetch": "^2.5.11",
-    "@types/ws": "^7.4.6",
-    "discord.js": "^12.5.3",
+    "@discordjs/collection": "^0.2.1",
     "events": "^3.3.0",
```

Comment:
what's this `events` for?

Reply:
The class `Manager` extends `EventEmitter`.

Reply:
but what's the point of installing `events`?

Reply:
The `EventEmitter` object is from the `events` module. Do you mean there is no need of event handelling in the module?

Reply:
Yes, **EventEmitter** is from the **[events](https://nodejs.org/dist/latest-v14.x/docs/api/events.html)** module which comes with NodeJS itself. I am asking what's the point of installing `events` from npm?

Reply:
Ahh i didn't noticed that.

Reply:
hmm didn't knew that it comes with nodeJS. Will change it